systers_portal/templates/403.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block title %} - Permission Denied{% endblock %}
+
+{% block content %}
+  
+  <div class="jumbotron jumbotron-custom">
+    <h1>403</h1>
+    <h1>Permission Denied</h1>
```

Comment:
Add a message for the user, something like "You don't have authorization to access the page". Also add a back button below the message, on onClick just say ```history.go(-1); return false;```. If you know a better way please go ahead.

Reply:
Should there be a back button on each of the error pages?

Reply:
in all pages.